contrib/sync-places.py

```diff
@@ -109,6 +109,13 @@ async def do_sync(session, args):
                     changed = True
 
             tags = config["places"][name].get("tags", {}).copy()
+            for k, v in tags.items():
+                if not isinstance(k, str):
+                    tags[str(k)] = v
+                    del(tags[k])
```

Comment:
There is a hole in this logic if both the key and value are not strings, in which case the `tags[k] = str(v)` will re-add the non-string key.

Reply:
oh! good catch. Should be fixed now

Reply:
`del()` needs to be before the assignment, otherwise it will remove the key if it's already a string :)